src/Parser/DeathKnight/Unholy/Modules/Features/CastEfficiency.js

```diff
@@ -39,12 +92,28 @@ class CastEfficiency extends CoreCastEfficiency {
     },
 
     {
-      spell: SPELLS.SUMMON_GARGOYLE,
-      category: CastEfficiency.SPELL_CATEGORIES.COOLDOWNS,
-      getCooldown: haste => 180, // TODO: needs to account for CoF
+      spell: SPELLS.SOUL_REAPER_TALENT,
+      category: CastEfficiency.SPELL_CATEGORIES.ROTATIONAL,
+      getCooldown: haste => 45,
+      isActive: combatant => combatant.hasTalent(SPELLS.SOUL_REAPER_TALENT),
       recommendedCastEfficiency: 0.90,
-      isActive: combatant => combatant.hasTalent(SPELLS.DEFILE_TALENT.id) || combatant.hasTalent(SPELLS.SOUL_REAPER_TALENT.id),
-      extraSuggestion: 'This is your main DPS cooldown.  Try to cast this off cooldown, keep in mind that if you are wearing <ItemLink id={ITEMS.TAKTHERITRIXS_SHOULDERPADS.id}/> that you make sure <SpellLink id={SPELLS.DARK_TRANSFORMATION.id}/> can be cast immediatly after.',
+    },
+
+    {
+      spell: SPELLS.BLIGHTED_RUNE_WEAPON_TALENT,
+      category: CastEfficiency.SPELL_CATEGORIES.COOLDOWNS,
+      getCooldown: haste => 60,
+      isActive: combatant => combatant.hasTalent(SPELLS.BLIGHTED_RUNE_WEAPON_TALENT),
+      recommendedCastEfficiency: .90,
+    },
+
```

Comment:
Might aswell do ``0.90`` like you've done everywhere else instead of ``.90`` all of a sudden

Reply:
Same with the epidemic talent right under